src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -2,17 +2,21 @@
   (:require [quo.components.animated.pressable :as animation]
             [quo.react-native :as rn]
             [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
             [quo2.components.notifications.activity-logs :as activity-logs]
+            [quo2.components.tabs.tabs :as tabs]
             [quo2.components.tags.context-tags :as context-tags]
             [quo2.foundations.colors :as colors]
             [reagent.core :as reagent]
             [status-im.constants :as constants]
             [status-im.i18n.i18n :as i18n]
             [status-im.multiaccounts.core :as multiaccounts]
-            [status-im.ui.components.topbar :as topbar]
             [status-im.utils.datetime :as datetime]
             [status-im.utils.handlers :refer [<sub >evt]]))
 
+(defonce selected-activity-type
+  (reagent/atom :activity-type/all))
```

Comment:
seems like not used? probably better to do not have it for now ? and implement later